src/ProofOfConcept.java

```diff
@@ -0,0 +1,66 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/**
+ * Creating bare bones methods for my playlist component.
+ */
+public class ProofOfConcept {
+
+    /**
+     * finds the song to be removed and removes it.
+     *
+     * @param playlist
+     * @param song
+     */
+    public static void removeSong(Queue<String> playlist, String song) {
```

Comment:
I think this is good enough for a proof of concept, but I am going to push you a little to make use of more interesting data types than String. For instance, you might store a Queue of Map, where the Map stores pairs of values like `(artist, ...)` and `(title, ...)`. And it scales well to support additional data later like the location of the song file, the location of the album art, etc. You might also provide playlist-level features like being able to name the playlist or rate the playlist.

Reply:
The consequence of the design above is that you may need to overload some of your methods. Like multiple addSong methods, some with barebones inputs (i.e., title, artist) and some with more complex inputs (e.g., title, artist, album, release date, etc.).